crates/core/src/network/io/egress_stream.rs

```diff
@@ -380,20 +382,20 @@ impl EgressStream {
                 msg_type,
                 sort_code,
                 service_tag,
-                span,
                 version,
+                notifier,
                 #[cfg(feature = "test-util")]
                     header: custom_header,
             })) => {
                 let mut header = Header::default();
-                self.fill_header(&mut header, span);
+                self.fill_header(&mut header, None);
```

Comment:
Are we primarily not sending the originating span to save some bytes? Are those the 8 bytes that we are saving? Couldn't it be helpful for tracing when trying to understand where messages came from?

Reply:
In fact, at the moment distributed tracing for the message fabric is disabled, but when we enable it back, I believe that the majority of the value will come from tracing RPC operation rather than unary messages. Unary messages are meant to be lightweight and fast (fire-and-forget).

In addition, this is not a one way decision, it doesn't break compatibility to add this back in a future version if we want.

It's worth noting that the saved bytes also come from swapping the String with the &'static str.